src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar :rofl: You are too funny.
I cannot see that on my Athlon XP 1700+, as he is totally quiet, no warnings, far better than your :wink: should i mention that i don't get output as well :rofl: 

We can make it gain faster, as the full hardward acceleration was not used.
- `ffmpeg64.exe -y -hwaccel auto -c:v h265_cuvid -i jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv -bufsize 6M -maxrate 6M -b:v 3M -c:v h264_nvenc -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark`
- `ffmpeg64.exe -y -hwaccel auto -c:v h265_cuvid -i jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv -bufsize 6M -maxrate 6M -b:v 3M -c:v h264_nvenc -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark`